src/gwt/src/org/rstudio/studio/client/common/spelling/TypoSpellChecker.java

```diff
@@ -191,33 +199,50 @@ private void updateIgnoredWordsIndex()
 
    private void loadDictionary()
    {
-      ExternalJavaScriptLoader.Callback loadTypoCallback = () -> {
-         typoNative_ = new TypoNative(
-            uiPrefs_.spellingDictionaryLanguage().getValue(),
-            null,
-            null,
-            null
-         );
-      };
-      new ExternalJavaScriptLoader(TypoResources.INSTANCE.typojs().getSafeUri().asString()).addCallback(loadTypoCallback);
-   }
+      String dictLanguage = uiPrefs_.spellingDictionaryLanguage().getValue();
+      String path = "/dictionaries/" + dictLanguage + "/" + dictLanguage;
```

Comment:
Generally it isn't safe to presume that RStudio is being served at the root path (it is often proxied so that it is accessed at e.g. `https://server/rstudio` instead of `https://server:8787/`.) If you need to use an absolute URL, look at `getHostPageBaseUrl`